deployment-templates/compute-engine/client/forseti-instance-client.py

```diff
@@ -15,6 +15,31 @@
 """Creates a GCE instance template for Forseti Security."""
 
 
+def get_patch_search_expression(forseti_version):
+    """Returns a glob expression matching all patches to the version of
```

Comment:
The first line of a doc string needs to be one sentence within 80 characters (for generating nice-looking pydocs).
So, please change this.
```
"""Returns a glob expression matching all patches of the given version.
```